src/geo.c

```diff
@@ -112,11 +112,17 @@ int extractLongLatOrReply(client *c, robj **argv, double *xy) {
 /* Input Argument Helper */
 /* Decode lat/long from a zset member's score.
  * Returns C_OK on successful decoding, otherwise C_ERR is returned. */
-int longLatFromMember(robj *zobj, robj *member, double *xy) {
+int longLatFromMember(robj *zobj, robj *member, double *xy, sds* err) {
```

Comment:
```suggestion
int longLatFromMemberOrReply(client *c, robj *zobj, robj *member, double *xy) {
```
A more common pattern we use is "validate" or reply. This makes it so we don't have to pass a buncch of stack allocated objects back and forth. If we pass in the client object, we can do all the logic and then also put the replies on the client object here. The replying code becomes centralized as well.

Reply:
yeah, its cleaner, i fixed it